br-webui/_includes/header.liquid

```diff
@@ -54,7 +55,16 @@
 					<button onclick="shutdown()" class="btn btn-error btn-sm navbar-btn">Shutdown</button>
 				</form>
 			</div><!--/.nav-collapse -->
+			<div>
+				<ul class="toggle">
+					<label class="switch">
+                        <input id='advanced-checkbox' type="checkbox">
+                        <span class="slidernew round"></span>
+					</label>
+					Enable Advanced Options
+				</ul>
+			</div>
 		</div>
 	</nav>
-
+	<div class="heading"></div>
```

Comment:
why is this necessary?

Reply:
the header was getting longer in height from the top thats why I added some space before the headings start so that the header doesnt cover up the main headings. Like Network Setup, Camera Setup. stuff like that.